File: Ex02.py
#Ex 02r
i = 0
fibonacci = 1
numero = int(input("Escreva o numero desejado: "))
while fibonacci < numero:
    i,fibonacci = fibonacci, i + fibonacci

if fibonacci == numero:
    print(f"O numero {numero} informado faz parte da sequencia fibonacci")
else:
    print(f"O numero {numero} não  faz parte da sequencia .")

File: exerciciosemJava/Ex02.java
package exerciciosemJava;

import java.util.Scanner;

public class Ex02 {
	public static void main(String[] args) {

		int i = 0;
		int fibonacci = 1;
		try (Scanner sc = new Scanner(System.in)) {
			System.out.print("Escreva o número desejado: ");
			int numero = sc.nextInt();
			while (fibonacci < numero) {
				int temp = fibonacci;
				fibonacci = i + fibonacci;
				i = temp;
			}
			if (fibonacci == numero) {
				System.out.printf("O número %d informado faz parte da sequência Fibonacci.", numero);
			} else {
				System.out.printf("O número %d não faz parte da sequência Fibonacci.", numero);
			}
		}
	}

}
